app/views/questions/_nav.html.erb

```diff
@@ -5,7 +5,7 @@
         <i class="nav-right-icons icon-right disabled"></i>
     <% else %>
         <a href="<%=
-            url_for :controller => 'questions', :action => 'next'
+            url_for :controller => 'questions', :action => 'next', :week_number => @week_number, :room_number => @room_number, :lo_id => @question.id
```

Comment:
Toto je velmi nestastny sposob, ako vyrobit URL. Netyka sa to ale tvojho Pull-Requestu @veronikagondova, bolo to tam uz pred tebou.